<launch>
  <!-- Image topic to subscribe to -->
  <arg name="image_topic" default="/zed/zed_node/point_cloud/cloud_registered"/>
  <!-- Cone topic to publish to -->
  <arg name="cone_topic" default="/perception/cone_image_map"/>
  <group ns="perception">
    <group ns="point_cloud">
      <!-- Launch thresholding node -->
      <node pkg="point_cloud" type="point_cloud" name="point_cloud" output="screen" required="true">
        <param name="image_topic" value="$(arg image_topic)"/>
        <param name="cone_topic" type="str" value="$(arg cone_topic)"/>
      </node>
    </group>
  </group>
</launch>
